src/arsenic/session.py

```diff
@@ -329,6 +336,11 @@ async def execute_script(self, script, *args):
             url="/execute", method="POST", data={"script": script, "args": list(args)}
         )
 
+    async def execute_async_script(self, script, *args):
+        return await self._request(
+            url="/execute_async", method="POST", data={"script": script, "args": list(args)}
```

Comment:
@derekmhh please `black` this file, that's what the build failure is about:
```
test-env/bin/black --check --line-length=88 --safe src tests setup.py 
 
would reformat /home/src/arsenic/session.py
 
Oh no! 💥 💔 💥
 
1 file would be reformatted, 24 files would be left unchanged.
 ```

The simplest way to go about it is to use `pre-commit` :)